File: 1013.py
x = input().split()
a, b, c = x

maiorAB = (int(a) + int(b) + abs(int(a) - int(b)))/2
m = (int(c) + (maiorAB) + abs(int(c) - (maiorAB)))/2
m = int(m)
print('{} eh o maior'.format(m))

File: CSharp/1013.cs
using System;
using System.Globalization;
using System.Linq;

class URI {
    static void Main(string[] args) {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        
        int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int x = a[0];
        int y = a[1];
        int z = a[2];
        
        int maior = ((x+y+Math.Abs(x-y))/2);
        maior = ((z+maior+Math.Abs(z-maior))/2);
        
        Console.WriteLine($"{maior} eh o maior");

    }
}
